PAPER/FIT_Paper_Ex1/Part_3.tex

```diff
@@ -150,57 +151,51 @@ \subsection{Exercise specifications}
 \subsection{Problem-solving methodology}
 
 In order to investigate the influence of using a \gls{FLM} rather than a
-\gls{FF} approach on fuel cycle simulations, the following experiment have been
-designed. Following the FIT project spirit, no code to code comparison will be
+\gls{FF} model on fuel cycle simulations, the following experiment have been
+designed. Following the FIT project principle, no inter-code comparison will be
 done. For each simulator used to solve this problem, fuel models will be
 compared within the same simulator, allowing to evaluate difference between two
 almost identical simulation: same simulator, reactor description, depletion
-algorithm, \ldots the only difference being the method used to build the fresh fuel.
+algorithm, \ldots The only difference being the method used to build the fresh fuel.
 
 The experiment consists in running a small fuel cycle calculation according to
 specifications defined above. We use a method that we call "Wide Parametric
-Sweeping" method. The principle of this method is to fill from a random 
-sampling method the plutonium isotopic space defined in the Table~\ref{
-Tab:PuVector}. The global set of plutonium isotopic vectors that will be run 
-is called the design of experiment. Each plutonium isotopic vector will be run 
-twice. One using a \gls{FLM} approach for which the plutonium fraction at 
-reactor \gls{BOC} directly depends on the plutonium isotopic composition. The 
-second run uses \gls{FF} approach for which the plutonium fraction at \gls{BOC}
-is always the same. 
+Sweep" method. The principle is to randomly populate the pre-defined isotopic
+space (see in the Table~\ref{tab:PuVector}. For each random plutonium
+composition the simulation will be ran twice, once using the \gls{FF} model,
+i.e. the fraction of plutonium in the fuel is fixed regardless to its isotopic
```

Comment:
There, the order was first FLM and then FF because some users may need FLM to calculate FF value. But detail....